SocketRocket/SRWebSocket.m

```diff
@@ -581,21 +586,26 @@ - (void)_initializeStreams;
             [SSLOptions setValue:[NSNumber numberWithBool:NO] forKey:(__bridge id)kCFStreamSSLValidatesCertificateChain];
         }
         
+        BOOL allowUntrustedSSLCertificates = self.allowUntrustedSSLCertificates;
+        
 #if DEBUG
-        [SSLOptions setValue:[NSNumber numberWithBool:NO] forKey:(__bridge id)kCFStreamSSLValidatesCertificateChain];
-        NSLog(@"SocketRocket: In debug mode.  Allowing connection to any root cert");
+        allowUntrustedSSLCertificates = YES;
 #endif
+
+        if (allowUntrustedSSLCertificates) {
+            [SSLOptions setValue:[NSNumber numberWithBool:NO] forKey:(__bridge id)kCFStreamSSLValidatesCertificateChain];
```

Comment:
Friendly note, might be easier to read `[NSNumber numberWithBool:NO]` if it was `@NO`! Wasn't you I know ^_^